app/src/main/java/com/buzzshelter/Controllers/EnterShelterActivity.java

```diff
@@ -0,0 +1,62 @@
+package com.buzzshelter.Controllers;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import com.buzzshelter.Model.Model;
+import com.buzzshelter.Model.Shelter;
+import com.example.tonyzhang.buzzshelter.R;
+
+/**
+ * Created by jeffr on 2/21/2018.
+ */
+
+public class EnterShelterActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_entershelter);
+
+        final EditText shelterName = findViewById(R.id.enterName);
```

Comment:
just curious. Why are these all final?

Reply:
For some reason the "final" modifiers were automatically added when I created the variable.
I have no idea why lol.